test/blackbox-tests/test-cases/actions/conc-multi-diff.t/run.t

```diff
@@ -0,0 +1,84 @@
+Illustration of the use of (conc ) in the context of diffing multiple files.
+
+Say we want to diff 3 files.
+  $ cat A
+  I am file A.
+  $ cat B
+  I am file B.
+  $ cat C
+  I am file C.
+
+We set up a (progn ) rule to diff all of them against their generated versions.
```

Comment:
```suggestion
We set up a (progn ) rule to diff all of them against their generated versions.
```

Should all these (progn ) & (conc ) be `(progn )` & `(conc )` since they appear in the code? I've seen it both ways in this document. If so, I can go back and change them all in this and the other one I just reviewed.

Reply:
Inside cram tests there's no interpretation of markdown syntax. So even if we wanted something to appear monospaced it wouldn't be possible.